src/components/popover/PopoverComponent.jsx

```diff
@@ -0,0 +1,20 @@
+import { Popover, Typography } from '@mui/material'
+import React from 'react'
+
+const PopoverComponent = ({anchorEl, open, setOpen, info, setProperties}) => {
+
+    const handleClose = () => {

```

Comment:
We can discuss whether to handle the close functionality here.
In fact, since _setProperties_ and _setOpen_ are provided as props, it might be a better approach to pass _handleClose_ as a prop to the Popover instead.